======
#the simple exception
a = 12;
try:
    print(a)
    print(b)
except Exception as errTest:
    print('有几把错了')
    print（err）
    
=============================================================
#this mathod can make your loop code continue


for i in range(1,10):
    try:
        if(i == 4):
            print('@')
            print(b)
        print(i)
    except Exception as err:
        print(err)
print('12 ')


#output result

1
2
3
@
name 'b' is not defined
5
6
7
8
9
12 
    